### ISSUE TITLE
Evolution Strategies as a Scalable Alternative to Reinforcement Learning

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#245]
## 一言でいうと

強化学習における遺伝的アルゴリズムの有用性を示した論文。Virtual batch normalizationを使うことで信頼性up。backprop、value functionなしで計算できてしかも並列化が可能。DQNに比べパフォーマンス＋学習にかかるまでの時間で優位との結果

### 論文リンク

https://arxiv.org/abs/1703.03864

### 著者/所属機関

Tim Salimans, Jonathan Ho, Xi Chen, Ilya Sutskever
OpenAI
